branding/pdf/entities.adoc

```diff
@@ -47,6 +47,9 @@
 :sle: SUSE Linux Enterprise
 :sleses: SUSE Linux Enterprise Server with Expanded Support
 :es: Expanded Support
+:slemicro: SLE Micro
```

Comment:
@keichwa this should not be needed AFAIK. Lana used either `:micro:` or `:microos:` at the doc?

Reply:
Correct, I had {slea}{nbsp}{micro} for SLE Micro, and {opensuse}{nbsp}{microos} for openSUSE MicroOS.